gst/tensor_ros_sink/tensor_ros_sink.c

```diff
@@ -0,0 +1,849 @@
+/**
+ * GStreamer
+ * Copyright (C) 2005 Thomas Vander Stichele <thomas@apestaart.org>
+ * Copyright (C) 2005 Ronald S. Bultje <rbultje@ronald.bitfreak.net>
+ * Copyright (C) 2018 Samsung Electronics Co., Ltd.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ */
+
+/**
+ * SECTION:element-tensor_ros_sink
+ *
+ * Sink element to publish a ROS topic for tensor stream
+ *
+ * @file    tensor_ros_sink.c
+ * @date    15 June 2018
+ * @brief   GStreamer plugin to publish a ROS topic for tensor stream
+ * @see     https://github.com/nnsuite/nnstreamer-ros
+ * @author  Wook Song <wook16.song@samsung.com>
+ * @bug     No known bugs except for NYI items
+ */
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include <nns_ros_bridge.h>
+#include <nnstreamer/tensor_common.h>
+
+#include "tensor_ros_sink.h"
+
+/**
+ * @brief Macro for debug mode.
+ */
+#ifndef DBG
+#define DBG (!self->silent)
+#endif
+
+/**
+ * @brief Macro for debug message.
+ */
+#define silent_debug(...) \
+    debug_print (DBG, __VA_ARGS__)
```

Comment:
Macro debug_print and err_print in tensor_common would be removed. Please change these.
nnstreamer issue 807(Revise logging mechanisms) and PR 875(Change log format)

Reply:
@jaeyun-jung  Changed as well.